README.md

```diff
@@ -3,11 +3,13 @@
 
 # Chasqui
 
-Chasqui is a simple, lightweight, persistent implementation of the publish-subscribe (pub-sub)
-messaging pattern for service oriented architectures.
+Chasqui is a simple, lightweight, persistent implementation of the
+publish-subscribe (pub-sub) messaging pattern for service oriented
+architectures.
```

Comment:
@jbgo (I'm going to be fairly blunt with my opinions - feel free to ignore and do whatever you think is best)

In general you use "simple" a lot in the README.  As an aside, you are not unique here as almost all libs seem to describe themselves as "simple" or "lightweight".  I think it is tiresome for gems to describe themselves this way.  (*We get it - you are simple - every gem aims to provide a simple API for a problem - that is the point of writing a gem.*)

I would find it refreshing if gems just described what they are/do and leave it to the users to judge how "simple" or "lightweight" they are.  In this case I would just say "Chasqui is a persistent implementation of the ..." or something that is just straight forward in describing what chasqui is.

This comment overall applies throughout the README too.

(again, this is just what I think and who cares about that, so you be the judge and do what makes you happy)